ee/tabby-webserver/src/service/cron/job.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::Result;
+use tokio_cron_scheduler::Job;
+use tracing::error;
+
+use crate::service::db::DbConn;
+
+pub(crate) async fn refresh_token_async(db_conn: DbConn) -> Result<Job> {
+    // job is run every 2 hours
+    let job = Job::new_async("0 0 1/2 * * * *", move |_, _| {
+        let db_conn = db_conn.clone();
+        Box::pin(async move {
+            let res = db_conn.delete_expired_token().await;
+            if let Err(e) = res {
+                error!("failed to delete expired token: {}", e);
+            }
+        })
+    })?;
+
+    Ok(job)
+}
+
+macro_rules! repository_job {
+    ($name:literal, $interval:expr) => {{
+        tokio_cron_scheduler::Job::new_repeated(
+            std::time::Duration::from_secs($interval),
+            |_, _| {
+                use tabby_common::schema::SchedulerRunMeta;
+
+                use crate::path::cron_job_runs_dir;
+
+                let start_time = chrono::Utc::now();
+
+                let exe = std::env::current_exe().unwrap();
+                let mut cmd = std::process::Command::new(exe);
```

Comment:
will subprocess's stdout / stderr being piped to current stdout / stderr (we don't want that)

Reply:
I tested locally, if use `inherit` here, like:
```
            cmd.stdout(Stdio::inherit())
                .stderr(Stdio::inherit())
                .args(["job::sync", "--job", &job_name]);
```

The job run output will display on the console, and the collected `output.stderr` & `output.stdout` are both empty.